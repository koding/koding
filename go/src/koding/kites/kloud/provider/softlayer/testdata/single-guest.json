{
  "resource": {
    "softlayer_virtual_guest": {
      "test": {
        "cpu": 1,
        "domain": "koding.com",
        "hourly_billing": true,
        "local_disk": true,
        "name": "test",
        "public_network_speed": 10,
        "ram": 1024,
        "region": "dal09"
      }
    }
  }
}
