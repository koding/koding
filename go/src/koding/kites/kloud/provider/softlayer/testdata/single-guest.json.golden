{
  "resource": {
    "softlayer_virtual_guest": {
      "test": {
        "cpu": 1,
        "domain": "koding.com",
        "hourly_billing": true,
        "image": "UBUNTU_14_64",
        "local_disk": true,
        "name": "test",
        "public_network_speed": 10,
        "ram": 1024,
        "region": "dal09",
        "ssh_keys": [
          12345
        ],
        "user_data": "***"
      }
    }
  }
}
